ovos_plugin_common_play/ocp/utils.py

```diff
@@ -2,14 +2,20 @@
 import shutil
 import tempfile
 from os import makedirs
-from os.path import basename, expanduser, isfile, join, dirname
+from os.path import basename, expanduser, isfile, join, dirname, exists
 
 from ovos_plugin_common_play.ocp.status import TrackState, PlaybackType
 from ovos_plugin_manager.ocp import StreamHandler
 from ovos_ocp_files_plugin.plugin import OCPFilesMetadataExtractor
 ocp_plugins = StreamHandler()
 
 
+def is_qtav_available():
+    return exists("/usr/include/qt/QtAV") or \
+           exists("/usr/lib/qt/qml/QtAV") or \
+           exists("/usr/lib/libQtAV.so")
```

Comment:
@AIIX which of these is directly needed? are there other paths we should check? these exist if installed via package manager in manjaro at least


other than this i think the PR can be merged